### PR TITLE
Update solid-logo-usage-guidelines.md

### PR DESCRIPTION
**Pull request by danwilkinsoncreative**
First redraft of logo/brand usage guidelines. Just missing colour/typography samples and graphics (logos, patterns).

### CODE REVIEW

File: solid-logo-usage-guidelines.md [hunk 1]
```diff
@@ -1,17 +1,176 @@
...
+
+# Logo
+
+There is one single Solid logo with two different applications:
+
 ![Solid Logo](https://avatars3.githubusercontent.com/u/14262490?v=3&s=200)
 
-# Colours
+**Contained**
+
```
Comment (danwilkinsoncreative) [inline solid-logo-usage-guidelines.md]: solid-logo-contained

File: solid-logo-usage-guidelines.md [hunk 2]
```diff
@@ -1,17 +1,176 @@
...
+**Header Blue** - #354866, R53 G72 B102
+Usage: all non header copy. Avoid using for anything else.
+
+**Body Grey** - #666666, R102 G102 B102
+Usage: all non header copy. Avoid using for anything else.
+
+**Warning Orange** - #FFA600, R255 G166 B0
+Usage: for warning messages only
+
+**Error Red** - #D0021B, R208 G2 B27
```
Comment (danwilkinsoncreative) [inline solid-logo-usage-guidelines.md]: error-red

File: solid-logo-usage-guidelines.md [hunk 3]
```diff
@@ -1,17 +1,176 @@
...
+Usage: for error messaging only
+___
+
+# Typography
+
+The Solid brand features two typefaces:
+
+**RALEWAY**
+
```
Comment (danwilkinsoncreative) [inline solid-logo-usage-guidelines.md]: Raleway

File: solid-logo-usage-guidelines.md [hunk 4]
```diff
@@ -1,17 +1,176 @@
...
+# Typography
+
+The Solid brand features two typefaces:
+
+**RALEWAY**
+ 
+Usage: headlines, sub headlines, buttons, pull quotes
+
+**Roboto**
+
```
Comment (danwilkinsoncreative) [inline solid-logo-usage-guidelines.md]: Roboto

File: solid-logo-usage-guidelines.md [hunk 5]
```diff
@@ -1,17 +1,176 @@
...
+Both fonts are available as [Google Fonts](https://fonts.google.com)
+___
+
+# Iconography
+
+Solid uses icons provided by [Font Awesome](https://fontawesome.com). This is not only for ease but also to ensure icons are as usable/accessible as possible.
+___
+
+# Gradiants
+
```
Comment (danwilkinsoncreative) [inline solid-logo-usage-guidelines.md]: solid-gradient

File: solid-logo-usage-guidelines.md [hunk 6]
```diff
@@ -1,17 +1,176 @@
...
+Solid has one gradient which combines the Primary (Royal Lavender) and Secondary (Deep Sky Blue, Cerulean Blue) colours in equal parts (33/33/33)
+
+Usage: as a background or as an overlay to a greyscale image. It should not be used in the logo, typography, icons or graphics.
+___
+
+# Graphic elements
+
+**Solid hex pattern**
```
Comment (danwilkinsoncreative) [inline solid-logo-usage-guidelines.md]: concentric-hex-pattern_2x

File: solid-logo-usage-guidelines.md [hunk 7]
```diff
@@ -1,17 +1,176 @@
...
+For app developers
+-
+Please use these guidelines for any instances where you are looking to reference Solid. In particular, please pay notice to the guidelines on logo and colour usage as well as how to refer to Solid and its key elements in text.
+
+For event organisers or communications/content creators
+-
+If you are looking to create Solid communications, collateral or content please follow all aspects of these guidelines. For guidance on creating user interfaces, videos, animation or sound please contact [Mitzi László](https://github.com/Mitzi-Laszlo) for further guidance.
+___
+
+# Values
```
Comment (raulrpearson) [inline solid-logo-usage-guidelines.md]: Should we consolidate this section with the [values document](https://github.com/solid/information/blob/master/solid-values.md)? @Mitzi-Laszlo

**Comment by Mitzi-Laszlo** [comment]
@danwilkinsoncreative this pull request was discussed in the W3C Solid Community call today and there was general enthusiasm and two minor comments: 1) to move away from google fonts 2) to move away from blue as a complementary colour.

File: solid-logo-usage-guidelines.md (hunk 7 above)
Comment (Mitzi-Laszlo) [reply solid-logo-usage-guidelines.md]: let's make a link so that it doesn't diverge

**Pull request merged**